dialer.go

```diff
@@ -78,6 +80,10 @@ type Dialer struct {
 	// dialerID uniquely identifies a Dialer. Used for monitoring purposes,
 	// *only* when a client has configured OpenCensus exporters.
 	dialerID string
+
+	// tokenSource supplies the OAuth2 token used for IAM DB Authn. If IAM DB
+	// Authn is not enabled, tokenSource is unused.
+	tokenSource oauth2.TokenSource
```

Comment:
This naming seems a little concerning because we're using a mere (non-nil) presence of this value to determine if IAMAuthN is enabled. Can we update the name a bit to better reflect this? Like maybe `iamTokenSource` or `iamDBAuthTs`? (the first one seems better)

Reply:
Good point. I agree.